docs/faq.md

```diff
@@ -29,7 +58,7 @@ undefined.” in the logs**
 
 This usually means that EarlGrey is being linked to more than once. Ensure that only the **Test Target**
 depends on *EarlGrey.framework* and EarlGrey.framework is embedded in the app under test (i.e. *$TEST_HOST*) from the
-test target's build phase.
+test target's built products via a Copy File(s) Build Phase.
 
 **Is there a way to return a specific element?**
```

Comment:
This should be much higher, probably the 2nd from the top. If you're modifying FAQs anyways, I'd suggest look at the existing entries and re-order them based on popularity with highest on the top.